### PR TITLE
Prep javascript operators and expressions - James Aguirre

### PR DESCRIPTION
**Pull request by james-aguirre**
Screenshot 2023-03-01 at 3 59 24 PM

### CODE REVIEW

File: prep-javascript-operators-and-expressions/prep-javascript-operators-and-expressions-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# prep-javascript-operators-and-expressions-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What data type is returned by an arithmetic operation?
```
Comment (BrettAlbright) [inline prep-javascript-operators-and-expressions/prep-javascript-operators-and-expressions-quiz-notes.md:7]: dont forget the quiz questions!

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
code looks good James, just don't forget the quiz question answers

File: prep-javascript-operators-and-expressions/prep-javascript-operators-and-expressions-quiz-notes.md (hunk 1 above)
Comment (james-aguirre) [reply prep-javascript-operators-and-expressions/prep-javascript-operators-and-expressions-quiz-notes.md:7]: @Zhiwutian Hey Brett I added the quiz answers, Is it okay to merge now?

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**